src/likelihoods/laplace.jl

```diff
@@ -65,26 +67,34 @@ function expected_auglik_precision(::LaplaceLikelihood, qΩ, ::AbstractVector)
     return (2 * tvmean(qΩ).ω,)
 end
 
-function logtilt(lik::LaplaceLikelihood, Ω, y, f)
+function logtilt(
+    lik::LaplaceLikelihood, Ω::TupleVector, y::AbstractVector, f::AbstractVector{<:Real}
+)
     return length(y) * (loggamma(1//2) - log(sqrtπ) - log(2 * lik.β)) +
```

Comment:
I'm assuming you're keeping this because it's a bit faster; should the code then have tests that check that the scalar version and 1-vector versions match up ?